client/Cargo.toml

```diff
@@ -28,6 +28,7 @@ casper-node = { version = "1.0.0", path = "../node" }
 casper-types = { version = "1.0.0", path = "../types", default-features = false, features = ["std"] }
 clap = "2"
 hex = { version = "0.4.2", features = ["serde"] }
```

Comment:
This should be a `dev` dependency, right?

Reply:
It compiles without it but I can add it to dev dependencies

Reply:
Well if it doesn't matter as a `dev` dependency either we can just remove it entirely.